### PR TITLE
[terra-functional-testing] Add visual regression service

### PR DESCRIPTION
**Pull request by emilyrohrbough**
### Summary

- This adds the [wdio-novus-visual-regression-service](https://github.com/Jnegrier/wdio-novus-visual-regression-service) to terra-functional-testing. This iteration is to add the contents as is with eslint fixes.

- Follow up changes will be made to clean up the service to better fit Terra's needs and to integrate the service into the v6 wdio configuration. 

- The original project used mocha for tests. I've pulled these tests in and will convert to jest soon to align with our testing strategy.

Apart of https://github.com/cerner/terra-toolkit/issues/401

@cerner/terra

[CONTRIBUTORS.md]: ../blob/main/CONTRIBUTORS.md
[License]: ../blob/main/LICENSE

### CODE REVIEW

File: .eslintignore [hunk 1]
```diff
@@ -1,3 +1,4 @@
 node_modules
 build
 reports
+lib
```
Comment (mjhenkes) [inline .eslintignore]: add a page break for each of these little red nope guys ⛔

File: .travis.yml [hunk 2]
```diff
@@ -17,7 +17,10 @@ jobs:
         - npm run danger
         - npm run lint
         - npm run jest
-    # Split out all the sizes to not eat up all the travis executors at once.
+
```
Comment (ryanthemanuel) [inline .travis.yml:20]: 😢

File: .travis.yml [hunk 3]
```diff
@@ -17,7 +17,10 @@ jobs:
...
+    - stage: mocha
```
Comment (ryanthemanuel) [inline .travis.yml]: 😢

File: .eslintignore (hunk 1 above)
Comment (emilyrohrbough) [reply .eslintignore]: fixed: https://github.com/cerner/terra-toolkit/pull/430/commits/c5f53239e40409db99c5768be53dfc102d5a4531

**Review by tbiethman — APPROVED** [review]
:+1:

**Review by mjhenkes — APPROVED** [review]
🤷‍♂️  lgtm?

File: packages/terra-functional-testing/src/terra-dev-site/doc/visual-regression-service/VisualRegressionService.doc.mdx [hunk 4]
```diff
@@ -0,0 +1,211 @@
+# Visual Regression Service
```
Comment (benbcai) [inline packages/terra-functional-testing/src/terra-dev-site/doc/visual-regression-service/VisualRegressionService.doc.mdx:1]: +1 on updating things that are not applicable to us.

**Comment by benbcai** [comment]
Disclaimer: I didn't look at every file. It is too much. I only skim thru to make sure the files are carried over and has no reference to "novus". It builds so lgtmt.

**Review by benbcai — APPROVED** [review]

**Comment by emilyrohrbough** [comment]
Thanks everyone! it was a hefty PR. Next one's should be easier!

**Pull request merged**